packages/sitecore-jss-nextjs/src/middleware/redirects-middleware.ts

```diff
@@ -173,9 +173,12 @@ export class RedirectsMiddleware extends MiddlewareBase {
     const redirects = await this.redirectsService.fetchRedirects(siteName);
     const tragetURL = req.nextUrl.pathname;
     const targetQS = req.nextUrl.search || '';
+    const language = this.getLanguage(req);
+    const modifyRedirects = structuredClone(redirects);
```

Comment:
I can't see if _structuredClone_ is defined somewhere

Reply:
This a new method for deep cloning object in array https://developer.mozilla.org/en-US/docs/Web/API/structuredClone